src/main/java/com/esri/samples/localserver/local_server_geoprocessing/LocalServerGeoprocessingController.java

```diff
@@ -135,12 +145,10 @@ protected void handleGenerateContours() {
         mapView.getMap().getOperationalLayers().add(mapImageLayer);
         btnGenerate.setDisable(true);
       } else {
-        Platform.runLater(() -> {

```

Comment:
yes, it usually does work without, that doesnt mean it is on the correct thread though. An API fix has gone in recently, so I should be able to remove this.